src/sentry/web/frontend/accounts.py

```diff
@@ -305,3 +307,79 @@ def list_identities(request):
         'AUTH_PROVIDERS': AUTH_PROVIDERS,
     })
     return render_to_response('sentry/account/identities.html', context, request)
+
+
+@csrf_protect
+@never_cache
+@login_required
+def show_emails(request):
+    user = request.user
+    primary_email = user.emails.get(email=user.email)
+    alt_emails = user.emails.all().exclude(email=primary_email.email)
+
+    email_form = EmailForm(user, request.POST or None,
+        initial={
+            'primary_email': primary_email.email,
+        },
+    )
+
+    if 'remove' in request.POST:
+        email = request.POST.get('email')
+        del_email = UserEmail.objects.filter(user=user, email=email)
+        del_email.delete()
+        return HttpResponseRedirect(request.path)
+
+    if email_form.is_valid():
+        old_email = user.email
+
+        email_form.save()
```

Comment:
it'd be worthwhile to add a basic test around this chunk of code (adding an email)